{"timestamp":"2025-05-23T12:57:22.895884","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-05-23T12:57:22.897806","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-05-23T12:57:23.149803","level":"info","event":"Executing: \n    CREATE TABLE IF NOT EXISTS options_data (\n        id VARCHAR(255) PRIMARY KEY,\n        open DECIMAL,\n        close DECIMAL,\n        low DECIMAL,\n        volume DECIMAL,\n        vwap DECIMAL,\n        timestamp DECIMAL,\n        transactions DECIMAL\n    );\n    ","logger":"airflow.task.operators.airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator"}
{"timestamp":"2025-05-23T12:57:23.155340","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-05-23T12:57:26.594712Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.597178Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.598414Z","level":"info","event":"Current task name:create_stream_data_table","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.602249Z","level":"info","event":"Dag name:options_data_pipeline","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.599557","level":"info","event":"Connection Retrieved 'postgres'","logger":"airflow.hooks.base"}
{"timestamp":"2025-05-23T12:57:26.606858","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-05-23T12:57:26.615383","level":"info","event":"Connection Retrieved 'postgres'","logger":"airflow.hooks.base"}
{"timestamp":"2025-05-23T12:57:26.617780","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OperationalError","exc_value":"connection to server on socket \"/var/run/postgresql/.s.PGSQL.5432\" failed: No such file or directory\n\tIs the server running locally and accepting connections on that socket?\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py","lineno":312,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":725,"name":"run"},{"filename":"/usr/local/lib/python3.12/contextlib.py","lineno":137,"name":"__enter__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":832,"name":"_create_autocommit_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py","lineno":173,"name":"get_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py","lineno":122,"name":"connect"}]}]}
{"timestamp":"2025-05-23T12:57:26.620322Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.621322Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.621470Z","level":"info","event":"Task:<Task(SQLExecuteQueryOperator): create_stream_data_table>","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.621631Z","level":"info","event":"Failure caused by connection to server on socket \"/var/run/postgresql/.s.PGSQL.5432\" failed: No such file or directory","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.621892Z","level":"info","event":"\tIs the server running locally and accepting connections on that socket?","chan":"stdout","logger":"task"}
{"timestamp":"2025-05-23T12:57:26.621999Z","level":"info","event":"","chan":"stdout","logger":"task"}
